starknet_py/tests/e2e/docs/migration_guide/test_account_comparison.py

```diff
@@ -41,7 +38,7 @@ async def test_account_comparison(
     # docs-3: start
     # Using execute method
 
-    await account_client.execute(call, max_fee)
+    # await account_client.execute(call, max_fee)
```

Comment:
🤔

Reply:
It is in the migration guide. I wanted to keep it unchanged

Reply:
Can we do `@pytest.mark.skip` maybe?